temporalcli/commands.workflow_test.go

```diff
@@ -114,3 +114,100 @@ func (s *SharedServerSuite) testSignalBatchWorkflow(json bool) *CommandResult {
 	}
 	return res
 }
+func (s *SharedServerSuite) TestWorkflow_Terminate_SingleWorkflowSuccess() {
```

Comment:
Is reason optional for terminate? It should be and should default to the same reason we have for batch.
Worth testing.

Reply:
It looks like our old code doesn't specify a reason for batch termination unless one is provided: https://github.com/temporalio/cli/blob/main/batch/batch_commands.go#L89 and https://github.com/temporalio/cli/blob/main/batch/batch_commands.go#L157

Is there a default reason we'd _like_ to use?

Reply:
https://github.com/temporalio/cli/blob/8ad817da1f6cde7e48446e5c4e1ab7c7d4e0c4a8/temporalcli/commands.workflow.go#L183

Reply:
Ah, I was looking at the old code :). I'll go add this to the non-batch code

Reply:
Make sure you fix the `--reason` help string too please.

Reply:
Done!